zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanData.java

```diff
@@ -247,8 +247,31 @@ public boolean isPageOther(HttpMessage msg) {
     }
 
     /**
-     * Tells whether or not the response has a status code between 400 and 499 (inclusive). Falls
-     * back to check {@code CustomPage.Type.NOTFOUND_404}. Checks if the message matches {@code
+     * Tells whether or not the response has a status code between 200 and 299 (inclusive), or
+     * {@code CustomPage.Type.OK_200}. Checks if the message matches {@code
+     * CustomPage.Type.NOTFOUND_404} or {@code CusotmPage.Type.ERROR_500} first, in case the user is
+     * trying to override something.
+     *
+     * @param msg the message that will be checked
+     * @return {@code true} if the message matches, {@code false} otherwise
+     * @since TODO Add version
+     */
+    public boolean isSuccess(HttpMessage msg) {
+        if (isCustomPage(msg, CustomPage.Type.NOTFOUND_404)
+                || isCustomPage(msg, CustomPage.Type.ERROR_500)) {
+            return false;
+        }
+        if (isCustomPage(msg, CustomPage.Type.OK_200)) {
+            return true;
+        }
+        return HttpStatusCode.isSuccess(
+                msg.getResponseHeader()
+                        .getStatusCode()); // TODO JavaDoc and dbl check then UnitTests.
```

Comment:
Remove?

Reply:
Thanks, was a placeholder when I had to walk away at one point.